### ISSUE TITLE
Disable bootstrap rom after it finishes.

### ISSUE THREAD

**Issue by brunodea** [brunodea/gebemula#6]
In the [wiki](http://gbdev.gg8.se/wiki/articles/Gameboy_Bootstrap_ROM#The_DMG_bootstrap), one can read:

> The CPU enters at $0 at startup, and the last two instructions of the code writes to a special register which disables the internal ROM page, thus making the lower 256 bytes of the cartridge ROM readable. 

I believe we need to replace the bootstrap rom data in memory with the first 0x100 bytes from the game rom. However, I may have misinterpreted this.

**Comment by brunodea** [comment]
Nope. Just disabled the bootstrap room and overwrote the memory up to 0x100 with the game rom bytes and it didn't work: tried to execute an instruction with opcode 0xF5, which doesn't exist.

--EDIT--
Ops! 0xF5 actually is a valid instruction: PUSH AF.

**Comment by brunodea** [comment]
After implementing the RST instruction, I realized it would jump back to the beginning of the Gameboy rom. Such instruction should be used with an immediate of 0x00, 0x08, 0x10, 0x18, 0x20, 0x28 or... So, I took a look at such places in a game rom and noticed that it contained valid instructions that ended up with a jump. So, I believe my previous test was wrong and that we actually have to implement this.
